### PR TITLE
New credit card section with cancel subscription flow

### PR DESCRIPTION
**Pull request by usirin**

### CODE REVIEW

**Comment by usirin** [comment]
cancel-sub

File: client/component-lab/CancelSubscriptionModal/CancelSubscriptionModal.coffee [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      <div>
+        <Label size="large" type="danger">
+          <strong style={fontWeight: 'bold'}>You are about to cancel your team’s subscription</strong>
+        </Label>
+      </div>
+      <div style={lineHeight: '20px'}>
+        <Label size="medium">
+          If you proceed, <strong>your subscription will be suspended
+          immediately</strong> and you and your team will NOT be able to use Koding until
+          you renew your subscription. Are you sure to cancel subscription?
```
Comment (sinan) [inline client/component-lab/CancelSubscriptionModal/CancelSubscriptionModal.coffee]:
```html
If you proceed, <strong>your subscription will be suspended
immediately</strong>; you and your team will <strong>NOT</strong> be able to use Koding until
you renew your subscription. Are you sure to cancel your subscription?
```

File: client/component-lab/CreditCard/CardInfo.coffee [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+{ Grid, Row, Col } = require 'react-flexbox-grid'
+
+CardNumber = require './CardNumber'
+CardIcon = require './CardIcon'
+CardDate = require './CardDate'
+
+module.exports = CardInfo = ({ number, brand, month, year, onToggleForm, formVisible }) ->
+
+  buttonTitle = if formVisible then 'Hide' else 'Change Card'
```
Comment (sinan) [inline client/component-lab/CreditCard/CardInfo.coffee]: `buttonTitle = if formVisible then 'Cancel' else 'Change Card'`

**Review by sinan — APPROVED** [review]
👍 minor language changes necessary.

**Review by gokmen — APPROVED** [review]

**Comment by gokmen** [comment]
last two builds aborted intentionally, there were only copy changes.

**Pull request merged**